### PR TITLE
Feat/toastr js

### PR DESCRIPTION
**Pull request by vasanthakumar-a**
Flash notification added to all the pages using toastr js package

### CODE REVIEW

File: app/controllers/communities_controller.rb [hunk 1]
```diff
@@ -41,26 +41,31 @@ def create
...
       redirect_to communities_path
     else
+      flash[:alert] = "Please fill all required Fields"
       render :new
     end
   end
 
   def update
     if @community.update(community_values)
+      flash[:notice] = "Community Updated Successfully"
```
Comment (iMagesh) [inline app/controllers/communities_controller.rb:54]: add all the success and error messages to local.yml file and use rails localize for that

File: app/controllers/reports_controller.rb [hunk 2]
```diff
@@ -1,10 +1,10 @@
 class ReportsController < ApplicationController
   def create
     report = Report.new (report_params)
-    if report.save 
-      flash[:notice] = "Thanks for your report. Your reporting helps make Reddit a better, safer, and more welcoming place for everyone; and it means a lot to us."
+    if report.save
+      flash[:notice] = "Thanks for your report"
```
Comment (iMagesh) [inline app/controllers/reports_controller.rb:5]: move all the messages to the local yml file

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -4,16 +4,30 @@
...
-
   <body>
     <%= render partial: "layouts/navbar" %>
     <%= yield %>
     <%= render partial: "layouts/footer" %>
+    <% if flash[:notice] %>
+      <script>
+        toastr.options = {
+            "closeButton": true
+        };
```
Comment (iMagesh) [inline app/views/layouts/application.html.erb:20]: why is this part duplicated? can we refactor it ?

Comment (vasanthakumar-a) [reply app/views/layouts/application.html.erb:20]: code refactored

**Pull request merged**